image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [519.508792, 0.000000, 342.919347, 0.000000, 520.188440, 225.774265, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.039802, -0.103055, 0.000360, 0.004439, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [518.542223, 0.000000, 346.322181, 0.000000, 0.000000, 522.988309, 225.804878, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
